tests/install/worker_ay_validation.pm

```diff
@@ -0,0 +1,23 @@
+use Mojo::Base 'openQAcoretest';
+use File::Basename;
+use testapi;
+use utils;
+
+sub run {
+    switch_to_root_console;
+    install_packages('autoyast2 libxml2-tools');
+    # get_var('AUTOYAST');
+    my $template_path = "https://raw.githubusercontent.com/os-autoinst/openQA/8f939dab54e878a9672057f5343582d43ff2760a/contrib/ay-openqa-worker.xml.erb";
+    #"https://raw.githubusercontent.com/os-autoinst/openQA/master/contrib/ay-openqa-worker.xml.erb";
+    my $profile_template = basename($template_path);
+    assert_script_run "wget $template_path";
+    # run-scripts=true
+
+    enter_cmd "yast2 autoyast check-profile filename=$profile_template output=result.xml run-erb=true";
+
+    assert_screen "check-profile-ok", timeout => 300;
```

Comment:
I'd make the needle name a bit more specific:
```suggestion
    assert_screen "check-autoyast-profile-ok", timeout => 300;
```

Reply:
Done and https://github.com/os-autoinst/os-autoinst-needles-openQA/pull/25 updated